examples/release.yaml

```diff
@@ -0,0 +1,13 @@
+# Image tags from the latest release
+cemanager:
+  image: docker.io/coderenvs/coder-service:1.16.1
+dashboard:
+  image: docker.io/coderenvs/dashboard:1.16.1
+envproxy:
+  image: docker.io/coderenvs/coder-service:1.16.1
+timescale:
+  image: docker.io/coderenvs/timescale:1.16.1
+envbox:
+  image: docker.io/coderenvs/envbox:1.16.1
+envbuilder:
+  image: docker.io/coderenvs/envbuilder:1.16.1
```

Comment:
These should just be fake image URLs, I doubt it matters if they are real or not. Saves us the effort of updating this every release.

I'd probably rename this to `images.yaml`, or merge into `kind.values.yaml`.

Reply:
fair point, will rename. I didn't want to merge it because my thinking was that we'd have other values files later, that share a common list 'o images